# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cf/link-conf-once.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Only link the directory if it's not yet done (otherwise it'll cause shared files).
	--jsaw

--- cf/Imake.rules.~1.10.~	2005-08-19 00:38:58.000000000 +0200
+++ cf/Imake.rules	2005-11-21 02:18:48.241697733 +0100
@@ -1740,8 +1740,10 @@
 		(cd Concat($(DESTDIR),npath/mdir); \			@@\
 		 tar xf -; exit 0); exit 0); \				@@\
 	  fi; \								@@\
-	  $(RM) -r rdir; \						@@\
-	  $(LN) Concat($${RELPATH},npath/mdir) ldir; \			@@\
+	  if [ ! -h rdir ]; then \					@@\
+	    $(RM) -r rdir; \						@@\
+	    $(LN) Concat($${RELPATH},npath/mdir) ldir; \		@@\
+	  fi; \								@@\
 	fi
 #else
 #define LinkConfDirectoryLong(mdir,cdir,rdir,ldir,opath,npath)
@@ -1785,8 +1787,10 @@
 	  if [ -f lfile -a ! -h lfile ]; then \				@@\
 	    cp -p lfile Concat($(DESTDIR),npath/cfile); \		@@\
 	  fi; \								@@\
-	  $(RM) -r lfile; \						@@\
-	  $(LN) Concat($${RELPATH},npath/cfile) lfile; \		@@\
+	  if [ ! -h rdir ]; then \					@@\
+	    $(RM) -r lfile; \						@@\
+	    $(LN) Concat($${RELPATH},npath/cfile) lfile; \		@@\
+	  fi; \								@@\
 	fi
 #else
 #define LinkConfFileLong(cfile,lfile,opath,npath)
